packages/linter/src/builders/lint/lint.impl.spec.ts

```diff
@@ -518,6 +518,6 @@ describe('Linter Builder', () => {
       force: false,
       maxWarnings: 1,
     });
-    expect(output.success).toBeFalsy();
+    expect(output.success).toBeTruthy();
```

Comment:
@wshowair Can you please change the 2nd the `warningCount` to `1` from `0` instead of changing the expectation? Also the test description seems that should be updated to something like `should be a failure if there are no errors, but there are more warnings than allowed by maxWarnings`